tests/test_playbooks/content_upload_ostree.yml

```
@@ -0,0 +1,67 @@
+---
+- hosts: localhost
+  collections:
+    - theforeman.foreman
+  gather_facts: false
+  vars_files:
+    - vars/server.yml
+  tasks:
+    - include: tasks/organization.yml
+      vars:
+        organization_state: present
+    - include: tasks/product.yml
+      vars:
+        product_state: present
+    - include: tasks/repository.yml
+      vars:
+        repository_state: absent
+        repository_name: "Test OStree Repository"
+        repository_content_type: "ostree"
+    - include: tasks/repository.yml
+      vars:
+        repository_state: present
+        repository_name: "Test OStree Repository"
+        repository_content_type: "ostree"
+
+- hosts: tests
+  collections:
+    - theforeman.foreman
+  gather_facts: false
+  vars_files:
+    - vars/server.yml
+  tasks:
+
+    - name: upload ostree archive
+      include: tasks/upload.yml
+      vars:
+        upload_src: "{{ playbook_dir }}/data/fixtures_small_repo.tar"
+        upload_repository: "Test OStree Repository"
+        upload_ostree_repository_name: "small"
+        expected_change: true
+    - name: upload ostree archive again, no change
```

```suggestion
      include_tasks: tasks/upload.yml
```

if you use `include_tasks` the `name` ("upload ostree archive again, no change") of the step is shown in the log, making the log nicer to read